core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5747,6 +5748,17 @@ public void subTestIntervalMonthFailsValidation() {
         .ok("INTERVAL '0' MONTH(0)");
   }
 
+  @Test void testSqlParserPosPlus() throws Exception {
+    final String sql = "insert into emps select * from emps";
+    final SqlNode sqlNode = getSqlParser(sql).parseStmt();
+    sqlNode.accept(new SqlShuttle() {
+      @Override public SqlNode visit(SqlIdentifier identifier) {
+        return new SqlIdentifier(identifier.names,
+            identifier.getParserPosition());
+      }
+    });
+  }
```

Comment:
how about add  some checks for the result of accept

Reply:
Thanks, add check for it.